rfcs/013-release_deployment_tracking/README.md

```diff
@@ -4,396 +4,211 @@
 
 ## Background
 
-We build container images as artifacts for deployment, they are labelled with the git ref at the code point the artifact was packaged.
+We have multiple projects that are themselves composed of multiple services, which can deploy into multiple environments (e.g. production/staging).
 
-Deploying container images to production at present moves all container images forward to the latest built version.
+Folk are regularly making changes, building updated services and deploying them into the appropriate environments. Keeping track of what is deployed where, by who and why is essential to working effectively.
 
-## Problem statement
+In general the build/release/deployment process can be described as follows:
 
-We wish to track multiple environments with container images at different versions, and have an audit trail for deployments to those environments.
+![Simple overview](simple_flow.png)
 
-A deployment mechanism to take a container image for a service within a project and run that in a given environment is deliberately not described here.
+More specifically including a high level view of infrastructure includes:
 
-SSM parameters are heavily favoured as they provide a data source for terraform.
+- A build environment to build artifacts, e.g. Travis
+- An artifact store to store things, e.g. ECR
+- A database to keep track of what should be released where e.g. DynamoDB
+- A Deployment environment to run things in, e.g. ECS
```

Comment:
I’m not sure this is adding anything over your diagram; if you want to keep it I'd flip it around to put the description first and the document-specific term second:

```suggestion
- A service that creates build artefacts from a given version of the codebase, e.g. creating a Docker image (a *build environment*)
- A store for the created artefacts, e.g. Docker images (an *artefact store*)
- An environment where services can run, e.g. ECS or Kubernetes (a *deployment environment*)
- A database that tracks what version of each application is running
```